core/domain/suggestion_services_test.py

```diff
@@ -1648,3 +1648,548 @@ def test_get_voiceover_application_from_model_with_invalid_type_raise_error(
+        self._assert_community_contribution_stats_is_in_default_state()
+
+        suggestion_services.reject_suggestion(
+            edit_state_content_suggestion.suggestion_id, self.reviewer_id,
+            'review message')
+
+        self._assert_community_contribution_stats_is_in_default_state()
+
+    def test_reject_edit_state_content_suggestions_does_not_change_the_counts(
+            self):
+        self._assert_community_contribution_stats_is_in_default_state()
+        edit_state_content_suggestion_1 = (
+            self._create_edit_state_content_suggestion())
+        edit_state_content_suggestion_2 = (
+            self._create_edit_state_content_suggestion())
+        self._assert_community_contribution_stats_is_in_default_state()
+
+        suggestion_services.reject_suggestions(
+            [
+                edit_state_content_suggestion_1.suggestion_id,
+                edit_state_content_suggestion_2.suggestion_id
+            ], self.reviewer_id, 'review message')
+
+        self._assert_community_contribution_stats_is_in_default_state()
+
+    def test_resubmit_edit_state_content_suggestion_does_not_change_the_counts(
+            self):
+        self._assert_community_contribution_stats_is_in_default_state()
+        edit_state_content_suggestion = (
+            self._create_edit_state_content_suggestion())
+        self._assert_community_contribution_stats_is_in_default_state()
+        suggestion_services.reject_suggestion(
+            edit_state_content_suggestion.suggestion_id, self.reviewer_id,
+            'review message')
+        self._assert_community_contribution_stats_is_in_default_state()
+        # Change the new_value of the html of the suggestion that got rejected
+        # so we can resubmit the suggestion for review.
+        resubmit_suggestion_change = edit_state_content_suggestion.change
+        resubmit_suggestion_change.new_value['html'] = 'new html to resubmit'
+
+        # Resubmit the rejected "edit state content" suggestion.
+        suggestion_services.resubmit_rejected_suggestion(
+            edit_state_content_suggestion.suggestion_id,
+            'resubmit summary message', self.author_id,
+            resubmit_suggestion_change)
+
+        self._assert_community_contribution_stats_is_in_default_state()
+
+    def test_create_question_suggestion_increases_question_suggestion_count(
+            self):
+        self._assert_community_contribution_stats_is_in_default_state()
+
+        self._create_question_suggestion()
+
+        stats = suggestion_services.get_community_contribution_stats()
+        self.assertEqual(stats.question_reviewer_count, 0)
+        self.assertEqual(stats.question_suggestion_count, 1)
+        self.assertDictEqual(
+            stats.translation_suggestion_counts_by_lang_code, {})
+        self.assertDictEqual(
+            stats.translation_reviewer_counts_by_lang_code, {})
+
+    def test_create_multi_question_suggestions_increases_question_count(self):
```

Comment:
Or: ``test_create_two_question_suggestions_increases_question_count`` (prefer specific test names to avoid "over-promising" on what's being tested, also 'multi' is more often used as a descriptor than a count).

Reply:
I actually don't think there's an issue with people wondering "why 2" since the test will be pretty clear about it. That being said, I can't come up with a stronger argument at the moment on why one is preferable over the other beyond my original point. I don't have a strong stance here.